app/controllers/ApprovalController.scala

```diff
@@ -40,10 +45,19 @@ class ApprovalController @Inject() (approvalService: ApprovalService, cc: Contro
 
   def get(id: String): Action[AnyContent] = Action.async { _ =>
     approvalService.getById(id).map {
-      case Right(process) => Ok(process)
+      case Right(approvalProcess) => Ok(Json.toJson(approvalProcess))
       case Left(Errors(NotFoundError :: Nil)) => NotFound(Json.toJson(NotFoundError))
       case Left(Errors(BadRequestError :: Nil)) => BadRequest(Json.toJson(BadRequestError))
       case Left(_) => InternalServerError(Json.toJson(InternalServiceError))
     }
   }
+
+  def listForAdminHomePage: Action[AnyContent] = Action.async { _ =>
```

Comment:
summaries ??

Reply:
I was thinking there may be a need for different summaries in future - so making it clear what this was for - but will change to make more generic

Reply:
have renamed approvalSummaryList